legal-api/src/legal_api/resources/__init__.py

```diff
@@ -24,6 +24,7 @@
 from flask_restx import Api
 
 from .business import API as BUSINESS_API
+from .document_signature import API as DOCUMENTS_API
```

Comment:
can you change the name here?
You're adding a documents.py, but exposing _document_signature_ as _documents_

Are we wrapping the minio service so that we upload docs through the API and retrieve them?

Reply:
No we are not wrapping minio service. legal-api provide a pre-signed url and the ui will upload directly to minio.

Reply:
we are going to open this directly to API users?

Reply:
Yes. Stamping should happen in entity-filer. right?

Reply:
Based on the stamping discussions we had in grooming, my understanding is that this will be done on the fly when the IA is applied and certify rules/memorandum need to be emailed out.  And also when downloading outputs from the ledger.

This is defined in the https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/7217 story.  Specifically, 
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/8368 and https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/8436 detail the work related to stamping.

Reply:
It says once filing is applied. Filing will be applied in entity filer when payment completed (entity-filer receive the notification of payment completed)